Modules/_bz2module.c

```diff
@@ -364,13 +364,16 @@ _bz2_BZ2Compressor_impl(PyTypeObject *type, int compresslevel)
         return NULL;
     }
 
+    // explicit fields initialization as PyObject_GC_New() does not change them
+    self->flushed = 0;
     self->bzs.opaque = NULL;
     self->bzs.bzalloc = BZ2_Malloc;
     self->bzs.bzfree = BZ2_Free;
     bzerror = BZ2_bzCompressInit(&self->bzs, compresslevel, 0, 0);
     if (catch_bz2_error(bzerror))
         goto error;
 
+    PyObject_GC_Track((PyObject *)self);
```

Comment:
```suggestion
    PyObject_GC_Track(self);
```

The function takes a `void *`.